pages/using_angular.md

```diff
@@ -31,7 +31,7 @@ Other available yarn/npm commands can be found in the `scripts` section of your
 
 ## Project Structure
 
-The JHipster client code can be found under `src/main/webapp`, and follows closely the  [John Papa Angular 2 style guide](https://github.com/johnpapa/angular-styleguide/blob/master/a2/README.md). Please read this guide first if you have any question on our application structure, file names, TypeScript conventions...
+The JHipster client code can be found under `src/main/webapp`, and follows closely the  [Angular style guide](https://github.com/johnpapa/angular-styleguide/blob/master/a2/README.md). Please read this guide first if you have any question on our application structure, file names, TypeScript conventions...
```

Comment:
There's probably an Angular official guide we should link to.

Reply:
https://angular.io/guide/styleguide

Reply:
Oh yes indeed, I thought we already were linking the official one !